Appsilon/mbaza
# Restyle migration

### Description
Stage 4 of #241. Merge #305 first. This PR is pure refactor; it introduces the last changes necessary for the automated tests to pass (mostly automatic restyling).

### How to test
1. The CI should pass.
2. Check if automated tests pass locally (`npm run lint`, `npx exec tsc` and `npm test`).
3. Review diff. Displaying a commit range excluding the automatic changes will be helpful :wink:
